backend/ml_model/repository/file_reader_multiple_models.py

```diff
@@ -2,17 +2,31 @@
 import os
 import pandas as pd
 import numpy as np
+import sys
 
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(project_root)
 
-class FileReaderMultiple:
+from utility import model_util
+from interfaces.file_reader_interface import FileReaderInterface
+
+
+class FileReaderMultiple(FileReaderInterface):
```

Comment:
Are all your classes following the correct dependency order?

Reply:
Yep.